internal/pkg/cli/svc_logs.go

```diff
@@ -193,6 +209,31 @@ func (o *svcLogsOpts) Execute() error {
 	return nil
 }
 
+func (o *svcLogsOpts) setPreviousStoppedTaskID() error {
+	env, err := o.configStore.GetEnvironment(o.appName, o.envName)
+	if err != nil {
+		return fmt.Errorf("get environment %s: %w", o.envName, err)
+	}
+	sess, err := o.sessProvider.FromRole(env.ManagerRoleARN, env.Region)
+	if err != nil {
+		return fmt.Errorf("get session %s: %w", o.envName, err)
+	}
+	svcDesc, err := o.newSvcDescriber(sess).DescribeService(o.appName, o.envName, o.name)
+	if err != nil {
+		return fmt.Errorf("describe serivce %s: %w", o.name, err)
+	}
+	if len(svcDesc.StoppedTasks) > 0 {
+		taskID, err := awsecs.TaskID(aws.StringValue(svcDesc.StoppedTasks[0].TaskArn))
+		if err != nil {
+			return err
+		}
+		o.taskIDs = append(o.taskIDs, taskID)
+	} else {
+		return fmt.Errorf("previously stopped task for the service %s: no logs available", o.name)
```

Comment:
I wonder if we shouldn't error and do a `log.Warningln("No previously stopped tasks found")` and print nothing to `stdout` 🤔

Reply:
Yeah that sounds good to me.